driver/labjack/writer_sink.cpp

```diff
@@ -251,15 +256,14 @@ std::vector<synnax::ChannelKey> labjack::WriteSink::get_state_channel_keys() {
 
 std::vector<synnax::ChannelKey> labjack::WriteSink::get_index_keys() {
     if (this->writer_config.channels.empty()) {
-        LOG(ERROR) << "[labjack.writer] No channels configured";
         return {};
     }
 
     std::set<synnax::ChannelKey> unique_keys;
     for (auto &channel: this->writer_config.channels) {
         auto [channel_info, err] = this->ctx->client->channels.retrieve(channel.state_key);
         if (err) {
-            LOG(ERROR) << "[labjack.writer] Failed to retrieve channel: " << channel.state_key;
+            this->log_err("Failed to retrieve channel.");
```

Comment:
This message gets returned to the caller, so it needs to be more detailed. What port is the problem in? User needs as much info as possible.

Reply:
Also address this for the other channel retrieval calls.

Reply:
resolved